package.json

```diff
@@ -12,7 +12,7 @@
   "license": "ISC",
   "main": "build.js",
   "scripts": {
-    "lint": "prettier --check data",
+    "lint": "prettier --check 'data/**/!(*.md)'",
```

Comment:
The alternative to this would be to change the prettier config to only apply the json-stringify on .json files and use regular prettier for other files.

However, our prettier config is still somewhat different from what I am used to and the resulting READM.md formatting is not great, so I think this is the better solution.

```diff
 % git diff
diff --git a/.prettierrc b/.prettierrc
index 27505a24..78e7d6df 100644
--- a/.prettierrc
+++ b/.prettierrc
@@ -1,6 +1,13 @@
 {
-    "parser": "json-stringify",
     "proseWrap": "never",
     "bracketSpacing": true,
-    "endOfLine": "lf"
+    "endOfLine": "lf",
+    "overrides": [
+        {
+            "files": "*.json",
+            "options": {
+                "parser": "json-stringify"
+            }
+        }
+    ]
 }
```